timed/reports/views.py

```diff
@@ -31,18 +31,18 @@ class YearStatisticViewSet(AggregateQuerysetMixin, ReadOnlyModelViewSet):
     filterset_class = ReportFilterSet
     ordering_fields = ("year", "duration")
     ordering = ("year",)
-    permission_classes = [
-        # internal employees or super users may read all customer statistics
-        (IsInternal | IsSuperUser) & IsAuthenticated
-    ]
+    permission_classes = (
+        (
+            # internal employees or super users may read all customer statistics
+            (IsInternal | IsSuperUser) & IsAuthenticated
+        ),
+    )
 
     def get_queryset(self):
         queryset = Report.objects.all()
         queryset = queryset.annotate(year=ExtractYear("date")).values("year")
         queryset = queryset.annotate(duration=Sum("duration"))
-        queryset = queryset.annotate(pk=F("year"))
-
-        return queryset
+        return queryset.annotate(pk=F("year"))
```

Comment:
Old pattern was more readable IMHO, consider disabling the rule, as it's not adding anything of value

Reply:
Well, one could as well go without intermediate variables at all, which would be even cleaner (and not violate the rule):

```py
return (
    queryset.annotate()
        .annotate()
        .annotate()
        .annotate()....
)
```